### ISSUE TITLE
People search returns duplicate results

### ISSUE THREAD

**Issue by jhoerr** [indiana-university/itpeople-functions-v2#14]
Search of a person by name/netid can return duplicate results. 

Example: `https://itpeople-api-test.azurewebsites.net/people?q=jhoer`

```
[
  {
    "netId": "jhoerr",
    "name": "John Hoerr",
    "nameFirst": "John",
    "id": 872,
    ...
  },
  {
    "netId": "jhoerr",
    "name": "John Hoerr",
    "nameFirst": "John",
    "id": 872,
    ...
  },
  {
    "netId": "jhoerr",
    "name": "John Hoerr",
    "nameFirst": "John",
    "id": 872,
    ...
  }
]
```

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/1151206/111797898-efd7b180-889f-11eb-9855-ed7af08d3174.png">

**Comment by jasonfrancis** [comment]
In acc0f7c I did try just adding a second `UnitMember` record for Ben to TestEntities so `PeopleTests.HasCorrectNumber()` could cover the scenario that was causing duplicates, but that broke assertions in a bunch of other tests, including the pact test, and I'm not touching those with a 10 foot pole. 😉️

**Comment by jasonfrancis** [comment]
> that broke assertions in a bunch of other tests

To be clear those aren't bad tests, they just depend on `TestEntities` having certain relationships, and adding any new `UnitMember` upsets their expectations.

**Comment by jasonfrancis** [comment]
Resolved with PR #18
